bot/resources/tags/cooldown.md

```diff
@@ -0,0 +1,22 @@
+**Cooldowns**
+
+Cooldowns are used in discord.py to rate-limit.
```

Comment:
Rate-limit what? Probably worth mentioning commands here.
```suggestion
Cooldowns are used in discord.py to rate-limit command execution.
```

Reply:
True, but the point of this is for custom cooldowns. Would !customcooldown or !customcd be a better tag?

Reply:
I think `customcooldown` would be a more suitable name (we tend to avoid abbreviations where possible), we can add a cooldown tag in future if we want.